### PR TITLE
added true memory

### PR DESCRIPTION
**Pull request by Pogogo007**
Added #42

### CODE REVIEW

**Comment by Pogogo007** [comment]
Im not an adept at c++ so a little cleanup would be nice

**Review by maximegmd — CHANGES_REQUESTED** [review]
Pass options to the function, don't create one like that, this can cause crashes.

**Comment by Pogogo007** [comment]
on it

**Comment by Pogogo007** [comment]
No idea if i did it correctly

File: cyberpunk_patch/src/pool_patch.cpp [hunk 1]
```diff
@@ -57,11 +57,11 @@ void RegisterPoolOptions(void* apThis, const char* acpName, uint64_t aSize)
         if (statex.ullTotalPhys)
         {
             const auto gigsInstalled = statex.ullTotalPhys / kScaler;
-            HMODULE AModule;
-            Options options(AModule);
+            Options options(mod);
```
Comment (PhantomGamers) [inline cyberpunk_patch/src/pool_patch.cpp:60]: This is what should be passed to the function

**Review by maximegmd — COMMENTED** [review]
Oh right this is called in the hook so it cant be passed this way, I guess it's time to make Options a singleton...

**Comment by Pogogo007** [comment]
> Oh right this is called in the hook so it cant be passed this way, I guess it's time to make Options a singleton...

no idea what that means but ill let ya do your thing

**Comment by yamashi** [comment]
Appreciate you trying, please do look at the solution I will post for this tomorrow, I will explain the commit why it has to be done that way

**Pull request closed without merging**